src/routes/console/project-[project]/auth/templates/strote.ts

```diff
@@ -0,0 +1,33 @@
+import type { Models } from '@appwrite.io/console';
+import { writable } from 'svelte/store';
+
+export const emailTemplate = writable<Models.EmailTemplate>({
```

Comment:
The filename seems to be a typo.. should be `store.ts`

Reply:
Nice catch :laughing: